apps/judicial-system/web/src/routes/Court/Indictments/Completed/Completed.tsx

```diff
@@ -112,30 +112,36 @@ const Completed: FC = () => {
         <Box marginBottom={5} component="section">
           <IndictmentCaseFilesList workingCase={workingCase} />
         </Box>
-        <Box marginBottom={5} component="section">
-          <SectionHeading
-            title={formatMessage(strings.criminalRecordUpdateTitle)}
-          />
-          <InputFileUpload
-            fileList={uploadFiles.filter(
-              (file) =>
-                file.category === CaseFileCategory.CRIMINAL_RECORD_UPDATE,
-            )}
-            accept="application/pdf"
-            header={formatMessage(core.uploadBoxTitle)}
-            buttonLabel={formatMessage(core.uploadBoxButtonLabel)}
-            description={formatMessage(core.uploadBoxDescription, {
-              fileEndings: '.pdf',
-            })}
-            onChange={(files) =>
-              handleCriminalRecordUpdateUpload(
-                files,
-                CaseFileCategory.CRIMINAL_RECORD_UPDATE,
-              )
-            }
-            onRemove={(file) => handleRemoveFile(file)}
-          />
-        </Box>
+        {workingCase.indictmentRulingDecision &&
+          [
```

Comment:
Consider creating a variable for this check as it's the exact same logic for whether or not to hide the next button in line 250